namui-user-config/Cargo.lock

```diff
@@ -0,0 +1,207 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This should not added in commit because namui-user-config is just lib. please remove it

Reply:
Ah!